Firestore/Example/Tests/Local/FSTMutationQueueTests.mm

```diff
@@ -255,36 +261,34 @@ - (void)testAllMutationBatchesAffectingDocumentKeys_handlesOverlap {
       FSTTestSetMutation(@"foo/baz", @{@"a" : @1}),
     ];
     FSTMutationBatch *batch1 =
-        [self.mutationQueue addMutationBatchWithWriteTime:[FIRTimestamp timestamp]
-                                                mutations:group1];
+        self.mutationQueue->AddMutationBatch([FIRTimestamp timestamp], group1);
 
     NSArray<FSTMutation *> *group2 = @[ FSTTestSetMutation(@"food/bar", @{@"a" : @1}) ];
-    [self.mutationQueue addMutationBatchWithWriteTime:[FIRTimestamp timestamp] mutations:group2];
+    self.mutationQueue->AddMutationBatch([FIRTimestamp timestamp], group2);
 
     NSArray<FSTMutation *> *group3 = @[
       FSTTestSetMutation(@"foo/bar", @{@"b" : @1}),
     ];
     FSTMutationBatch *batch3 =
-        [self.mutationQueue addMutationBatchWithWriteTime:[FIRTimestamp timestamp]
-                                                mutations:group3];
+        self.mutationQueue->AddMutationBatch([FIRTimestamp timestamp], group3);
 
     DocumentKeySet keys{
         Key("foo/bar"),
         Key("foo/baz"),
     };
 
-    NSArray<FSTMutationBatch *> *expected = @[ batch1, batch3 ];
-    NSArray<FSTMutationBatch *> *matches =
-        [self.mutationQueue allMutationBatchesAffectingDocumentKeys:keys];
+    std::vector<FSTMutationBatch *> expected{batch1, batch3};
+    std::vector<FSTMutationBatch *> matches =
+        self.mutationQueue->AllMutationBatchesAffectingDocumentKeys(keys);
 
-    XCTAssertEqualObjects(matches, expected);
+    [self assertVector:matches matchesExpected:expected];
   });
 }
 
 - (void)testAllMutationBatchesAffectingQuery {
   if ([self isTestBaseClass]) return;
 
-  self.persistence.run("testAllMutationBatchesAffectingQuery", [&]() {
+  self.persistence.run("testAllMutationBa˛tchesAffectingQuery", [&]() {
```

Comment:
"˛" is an odd character...

Reply:
Weird, not sure how that got there. Removed.